reference/5.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md

```diff
@@ -45,12 +45,13 @@ This cmdlet was introduced in Windows PowerShell 3.0.
 Get the latest feeds from the PowerShell team's blog.
 
 ```PowerShell
-Invoke-RestMethod -Uri https://blogs.msdn.microsoft.com/powershell/feed/ |
+PS C:\>Invoke-RestMethod -Uri https://blogs.msdn.microsoft.com/powershell/feed/ |
     Format-Table -Property Title, PubDate
 ```
 
+Output:
```

Comment:
This doesn't comply with platyPS [schema](https://github.com/PowerShell/platyPS/blob/master/platyPS.schema.md#version-200) (which causes failure in AppVeyor CI)

```
## EXAMPLES
// for every example
    ### {Example Name}

    {{Example introduction text}}

    // one or more times, codesnippet
    // it's useful to put the ```powershell code
    // before the plain text command exectution output
        ```{Syntax language, i.e. PowerShell or nothing for plain text}
        {{Example body}}
        ```

    {{Example remarks}} // not a mandatory, i.e. TechNet articles don't use remarks

```

Note that text is not allowed between codesnippets.